wgsl/index.bs

```diff
@@ -11751,6 +11773,10 @@ but a value may infer the type.
     <td>Description
     <td>Returns the natural logarithm of `e`.
     [=Component-wise=] when `T` is a vector.
+  <tr>
+    <td>
+    <td>
+Note: The result is not mathematically meaningful if `e` &lt 0.
```

Comment:
Right.  log(0.0f) can be negative infinity.  That is meaningful in some floating point enviroments.

But please fix the rendering: it needs a semicolon `&lt;`
Also insert blank line